orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/KmsConnectionService.java

```diff
@@ -28,7 +28,9 @@ public KmsConnectionService(ConfigurationService configurationService) {
     public KmsConnectionService(
             Optional<String> localstackEndpointUri, String awsRegion, String tokenSigningKeyId) {
         if (localstackEndpointUri.isPresent()) {
-            LOG.info("Localstack endpoint URI is present: " + localstackEndpointUri.get());
+            LOG.info(
+                    String.format(
+                            "Localstack endpoint URI is present: %s", localstackEndpointUri.get()));
```

Comment:
Does this work? Won't work where we're passing in an exception
```suggestion
            LOG.info("Localstack endpoint URI is present: {}", localstackEndpointUri.get());
```